notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>Be more specific that the user is testing the training the model within the container image locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>Will do</p>